content/kubernetes/re-databases/ingress_routing_with_istio.md

```diff
@@ -0,0 +1,155 @@
+---
+Title: Configure Istio for external routing
+linkTitle: Istio ingress routing
+description: Configure Istio as an ingress controller for access to your Redis Enterprise databases from outside the Kubernetes cluster. 
+weight: 25
+alwaysOpen: false
+categories: ["Platforms"]
+aliases: [
+    /kubernetes/re-databases/ingress_routing_with_istio.md,
+    /kubernetes/re-databases/ingress_routing_with_istio/,
+]
+---
+
+Redis Enterprise for Kubernetes version `6.2.8-tbd` introduces the ability to use an Istio ingress gateway as an alternative to NGINX or HaProxy ingress controllers.
+
+Istio can also understand ingress resources, but using that mechanism takes away the advantages and options that the native Istio resources provide. Istio offers its own configuration methods using custom resources.
+
+To configure Istio to work with the Redis Kubernetes operator, we will use two custom resources: a `Gateway` and a `VirtualService`. Then you'll be able to establish external access to your database.
+
+## Install and configure Istio for Redis Enterprise
+
+1. [Download](https://istio.io/latest/docs/setup/getting-started/) and [install](https://istio.io/latest/docs/setup/getting-started/) Istio (see instructions from Istio's [Getting Started](https://istio.io/latest/docs/setup/getting-started/) guide).
```

Comment:
The same "Getting Started" link appears three times here. Maybe limit the link to a single occurrence or provide more specific links for download and install?